EventVisualisation/Workflow/src/O2DPLDisplay.cxx

```diff
@@ -174,17 +185,21 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
   dataRequest->requestTracks(srcTrk, useMC);
   dataRequest->requestClusters(srcCl, useMC);
 
-  if (cfgc.options().get<bool>("read-from-files")) {
-    InputHelper::addInputSpecs(cfgc, specs, srcCl, srcTrk, srcTrk, useMC);
+  auto filterITSROF = cfgc.options().get<bool>("filter-its-rof");
+
+  if (filterITSROF) {
+    dataRequest->requestIRFramesITS();
   }
 
+  InputHelper::addInputSpecs(cfgc, specs, srcCl, srcTrk, srcTrk, useMC);
```

Comment:
Here you request reading data from the files, this can be done only if you are running display in a standalone mode over the data produced in advance. Using InputHelper should be conditioned:
```
if (!cfgc.options().get<bool>("disable-root-input")) {
  InputHelper::addInputSpecs(cfgc, specs, srcCl, srcTrk, srcTrk, useMC);
}
```
and `--disable-root-input` must be  requested when o2-eve-display runs with the direct input from the reco workflows., please add it to https://github.com/AliceO2Group/AliceO2/blob/6fb55ea45025b352b710e2bebdcc28b4cadbe556/prodtests/full-system-test/dpl-workflow.sh#L499

Reply:
Right, I forgot that. Then you just need to update the `dpl-workflow.sh`.